doc/asciidoc/consumer/index.adoc

```diff
@@ -313,6 +315,145 @@ By default no properties will be attached to the edge nodes but you can specify
 
 |===
 
+====== CUD File Format
+
+The CUD file format is JSON file that represents Graph Entities (Nodes/Relationships) and how to mange them in term
```

Comment:
I'm not 100% sure how we manage the differentiation between this and the pattern based extraction?